radix-engine/src/blueprints/epoch_manager/validator.rs

```diff
@@ -525,6 +562,16 @@ impl SecurifiedAccessRules for SecurifiedValidator {
                     )))),
                 ),
             ),
+            (
+                // TODO(resolve during review): we should probably rename it, but this is actually
+                // 100% precisely what this method does... Alternatively, we can make it public?
+                // (maybe there are cases where someone wants to simply boost the SU value?)
```

Comment:
I'd actually think about just removing this method call and accessing the state directly in the future (we don't have that api available now). 

I'd suggest an intent-based name rather than a logical name so something like `reward_validator`.

Reply:
cool idea; I went in this direction, but ended up with `apply_reward` 👍 (we technically don't reward the validator, but the owner, the stakers, etc).